frontend/app/src/locales/en.json

```diff
@@ -102,7 +102,24 @@
     },
     "title": "A supported asset update is available",
     "total_changes": "This update contains {changes} asset changes.",
-    "up_to_version": "Only update up to version"
+    "up_to_version": "Only update up to version",
+    "restore": {
+      "title": "Restore default assets database",
+      "subtitle": "Delete custom assets and restore default assets to their initial state. Caution: This operation might affect other users using the same database for assets.",
+      "action": "Restore",
+      "delete_confirmation": {
+        "title": "Restore default assets database",
+        "message": "Do you really want to restore your local assets database? This will keep the assets that you manually added and were not known to rotki before and reset all others to their original state."
```

Comment:
I think I am getting this confirmation at the hard reset ... which is wrong.

https://user-images.githubusercontent.com/1658405/130145077-3236cb31-1689-422a-b1e4-eb8931ba5da6.mp4